Fix misc issues that crop up with newer gcc versions.

--- yiff-2.14.2/yiff/Makefile
+++ yiff-2.14.2/yiff/Makefile
@@ -111,7 +111,7 @@
 #   farther above.
 #
 #LIB = -lm -lkmid
-LIB = -lm
+LIB = -lm -lstdc++
 
 # Library Directories:
 #
--- yiff-2.14.2/yiffconfig/yccb.c
+++ yiff-2.14.2/yiffconfig/yccb.c
@@ -168,13 +168,13 @@
 	{
 	    int status = CDialogGetResponse(
 		"Discard changes?",
-"Changes have not been saved, are you sure you\n\
-want to discard those changes?",
-"There are changes made to the configuration which\n\
-have not been saved. If you say 'Yes' then any\n\
-changes you have made will be lost. If you want\n\
-to preserve those changes then say 'No' and\n\
-continue on with the configuration.",
+"Changes have not been saved, are you sure you\n"\
+"want to discard those changes?",
+"There are changes made to the configuration which\n"\
+"have not been saved. If you say 'Yes' then any\n"\
+"changes you have made will be lost. If you want\n"\
+"to preserve those changes then say 'No' and\n"\
+"continue on with the configuration.",
 		CDIALOG_ICON_WARNING,
 		CDIALOG_BTNFLAG_YES | CDIALOG_BTNFLAG_NO |
 		CDIALOG_BTNFLAG_HELP,
--- yiff-2.14.2/yiffutils/ymixer.c
+++ yiff-2.14.2/yiffutils/ymixer.c
@@ -23,26 +23,26 @@
  */
 static void print_help(void)
 {
-	printf("\
-Usage: ymixer [mixer] [value1] [value2] [value3] [value4] [options]\n\
-\n\
-    [mixer] specifies the mixer channel device name, running without\n\
-    any arguments prints a list of available mixer names.\n\
-\n\
-    [value#] specify the percent values (from 0.0%%, 100.0%%) to be set.\n\
-    If no [value#] is specified then the current value is printed.\n\
-\n\
-    [options] can be any of the following:\n\
-\n\
-        --recorder <address:port>    Specify which Y server to connect to.\n\
-\n\
-    Return values:\n\
-\n\
-        0       Success.\n\
-        1       General error.\n\
-        2       Cannot connect to Y server error.\n\
-        3       Systems error.\n\
-\n"
+	printf("\n"\
+"Usage: ymixer [mixer] [value1] [value2] [value3] [value4] [options]\n"\
+"\n"\
+"    [mixer] specifies the mixer channel device name, running without\n"\
+"    any arguments prints a list of available mixer names.\n"\
+"\n"\
+"    [value#] specify the percent values (from 0.0%%, 100.0%%) to be set.\n"\
+"    If no [value#] is specified then the current value is printed.\n"\
+"\n"\
+"    [options] can be any of the following:\n"\
+"\n"\
+"        --recorder <address:port>    Specify which Y server to connect to.\n"\
+"\n"\
+"    Return values:\n"\
+"\n"\
+"        0       Success.\n"\
+"        1       General error.\n"\
+"        2       Cannot connect to Y server error.\n"\
+"        3       Systems error.\n"\
+"\n"
 	);
 }
 
